### DISCUSSION TITLE
Storing Date and Time Data in Exports of Notes

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#4025] [Ideas]
Hello,

A few months ago, my system broke down, and I had to, very arduously and with luck, export all my notes into a zip file using the `Export` function, in order to load them into Trilium after reinstalling Ubuntu.

I thought that this was great, but I just realized that it might be missing something. Specifically, I was hoping that the information about the dates on which I wrote several poems would be preserved. But I don't think that happened. Instead, it said that the dates I wrote those poems were all on the same date: the date I restored those notes. That information is lost, and my heart aches, as I have lost an important aspect of my history, but it would be nice if this could be changed for the future, so that reimporting notes would preserve that information.

Thanks for reading this. I hope you and yours are well.

**Comment by zadam** [comment]
Hello, from the system point of view, an imported note is truly a "new" note, it has a new identity and therefore also new creation date. Somehow it seems to me that preserving the original created date is less correct, although I can understand your point of view as well.

**Reply by protean-onion** [reply]
Could you add an option for it when we export notes? An option would be nice.

**Comment by jwhonce** [comment]
Tools like cpio do have options like --preserve-modification-time when creating files. 

I have a local server that I am running a containerized version of Trilium server to sync my laptops against. I wanted a disaster recovery planned before integrating a PKM into my work life.

**Reply by zadam** [reply]
Well, disaster recovery strategy is backing up the document.db

**Reply by jwhonce** [reply]
I should do that as well. Paranoia is a good thing. :-)
